src/main/java/org/prebid/server/bidder/imds/ImdsBidder.java

```diff
@@ -20,52 +20,57 @@
 import org.prebid.server.json.JacksonMapper;
 import org.prebid.server.proto.openrtb.ext.ExtPrebid;
 import org.prebid.server.proto.openrtb.ext.request.ExtRequest;
-import org.prebid.server.proto.openrtb.ext.request.synacormedia.ExtImpSynacormedia;
-import org.prebid.server.proto.openrtb.ext.request.synacormedia.ExtRequestSynacormedia;
+import org.prebid.server.proto.openrtb.ext.request.imds.ExtImpImds;
+import org.prebid.server.proto.openrtb.ext.request.imds.ExtRequestImds;
 import org.prebid.server.proto.openrtb.ext.response.BidType;
 import org.prebid.server.util.BidderUtil;
 import org.prebid.server.util.HttpUtil;
+import org.prebid.server.version.PrebidVersionProvider;
 
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 
-public class SynacormediaBidder implements Bidder<BidRequest> {
+public class ImdsBidder implements Bidder<BidRequest> {
 
-    private static final TypeReference<ExtPrebid<?, ExtImpSynacormedia>> SYNACORMEDIA_EXT_TYPE_REFERENCE =
+    private static final TypeReference<ExtPrebid<?, ExtImpImds>> IMDS_EXT_TYPE_REFERENCE =
             new TypeReference<>() {
             };
 
     private final String endpointUrl;
+    private final String prebidVersion;
     private final JacksonMapper mapper;
 
-    public SynacormediaBidder(String endpointUrl, JacksonMapper mapper) {
+    public ImdsBidder(String endpointUrl, PrebidVersionProvider prebidVersionProvider, JacksonMapper mapper) {
         this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.prebidVersion = Objects.requireNonNull(prebidVersionProvider.getNameVersionRecord());
```

Comment:
Why not pass version directly? You are injecting useless dependency.

Reply:
Resolved. The IDE erroneously added that and I didn't catch it.

Reply:
What I meant - pass version string directly instead of passing version provider as an argument to constructor. It's simpler.